lib/suse/connect/api.rb

```diff
@@ -60,36 +60,17 @@ def list_subscriptions
         make_paginated_request(:get, "#{connect_api}/organizations/subscriptions")
       end
 
-      def forward_system_activations(system)
-        params = {
-          login: system.login,
-          password: system.password,
-          hostname: system.hostname,
-          regcodes: [],
-          products: generate_product_listing_for(system),
-          hwinfo: generate_hwinfo_for(system),
-          last_seen_at: system.last_seen_at
-        }
-
-        make_single_request(
-          :post,
-          "#{connect_api}/organizations/systems",
-          { body: params.to_json }
-        )
-      end
-
       def send_bulk_system_update(systems, system_limit = nil)
         system_limit ||= BULK_SYSTEM_REQUEST_LIMIT
-        last_response = nil
         systems.each_slice(system_limit) do |batched_systems|
```

Comment:
Following the path on how this function is called, at this point the variable `systems` would still be an ActiveRecord relationship. This means we could also use `system.in_batches(of: system_limit)` here.

If I am not mistaken, `each_slice` is from the `Enumerable` API and would load all objects before slicing it. Since we don't know how many systems a RMT server can manage, maybe we should avoid loading them all at once.